.gitignore

```diff
@@ -1,7 +1,9 @@
-# See https://help.github.com/ignore-files/ for more about ignoring files.
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
 # dependencies
 /node_modules
+/pnp
+.pnp.js
```

Comment:
Suggestions from CRA v4. I think this relates to yarn v2.

Reply:
💡 https://next.yarnpkg.com/features/pnp